### ISSUE TITLE
0.20.4 updated package.json not commited

### ISSUE THREAD

**Issue by xrmx** [apache/incubator-superset#3738]
Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [x] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar

### Superset version

0.20.4

### Expected results

when i install from master 0.20.4 is installed

### Actual results

when i install from master 0.20.1 is installed 

### Steps to reproduce

python setup.py install

**Comment by xrmx** [comment]
Same for 0.20.5

**Comment by mistercrunch** [comment]
Yeah I started stamping the version on the release branch itself, though master should say something like 0.20.dev or something making it clear that it's not an actual release.

I started that since `master` is a protected branch and I cannot just push a version number when I cut the version. I'd have to go through a PR/build cycle to do that, and many times I want to release a previous SHA anyways, and I can't just change history and squeeze my alteration right after the target SHA...

Open for suggestions here. I'd go with `trunk` but I think pypi will trip om that
